modules/bot.js

```diff
@@ -132,6 +132,10 @@ var Bot = BotBase.extend(function () {
                         this.postMessage('Support Room:\nhttps://groupme.com/join_group/11752971/5jvG41');
                     }
 
+                    if (/^dirty martini$/.test(txt)) {
+                        this.postMessage('http://greygoose.com/GreyGoose/media/images/recipes/feature/recipe-Original-DIRTY-MARTINI-feature.jpg');
```

Comment:
i think this will post the link as msg and wouldn't use the img attachment feature. 
for image attachment use: this.postMessage('',imageUrl)